gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Automating and Scaling Vex Generation"
-date: 2023-03-06T09:00:00Z
+date: 2023-03-01T09:00:00Z
```

Comment:
Actually can we make this more like March 5th instead of 1st? That's closer to the actual date.